src/resources/Cookie/CookieMethods.ts

```diff
@@ -0,0 +1,15 @@
+export function getCookie(cname: string) {
+  const name = cname + '=';
+  const decodedCookie = decodeURIComponent(document.cookie);
+  const ca = decodedCookie.split(';');
+  for (let i = 0; i < ca.length; i++) {
+    let c = ca[i];
```

Comment:
Couldn't this while underneath be replaced by a trim()?
```suggestion
    let c = ca[i].trim();
```

Reply:
Maybe.. Not sure how much time would be saved in this case, and there is an argument to be made about readability, but this _is_ only nitpicking so it doesn't really matter that much.
It's fine as it is 🙂